commonmark/src/test/java/org/commonmark/test/UsageExampleTest.java

```diff
@@ -10,11 +12,20 @@
 public class UsageExampleTest {
 
     @Test
-    public void one() {
+    public void one() throws IOException {
         Parser parser = Parser.builder().build();
         Node document = parser.parse("This is *Sparta*");
         HtmlRenderer renderer = HtmlRenderer.builder().escapeHtml(true).build();
         assertEquals("<p>This is <em>Sparta</em></p>\n", renderer.render(document));
     }
+    
+    @Test
+    public void two() throws IOException {
+        Parser parser = Parser.builder().build();
+        StringReader sr = new StringReader("This is *Sparta*");
```

Comment:
Let's add a test that uses a real file, e.g. `spec.txt`. You can do `UsageExampleTest.class..getResourceAsStream("/spec.txt")` to get an InputStream for the file. Compare it to rendering the same file using a `String` as an input.

Also, this class is for the examples that are used in `README`. Could you move it to a new `ParserTest` instead?